.github/workflows/deploy-production.yml

```diff
@@ -1,8 +1,5 @@
 name: Deploy Production version
-on:
-  push:
-    branches:
-      - production
+on: [pull_request]
```

Comment:
Thanks, @gu-stav! I wouldn't touch this for now, though. If I understand properly, this would trigger a deploy to production with any PR. For now we still want to control the push to production. (cc @derrickmehaffy)

Reply:
Sorry, that was a mistake and an old thing lingering around. Will remove 👍🏼

Reply:
No worries, I still find it interesting :)